### PR TITLE
*_info - improve RETURN block of docs

### LINKED ISSUE #843: *_info modules missing/inaccurate RETURN data
### Summary

There are quite a few missing/inaccurate/not detailed enough RETURN values in the info modules. These should be up to date.

### Issue Type

Documentation Report

### Component Name

*_info

### Ansible Version

```console (paste below)
$ ansible --version

```

### Collection Versions

```console (paste below)
$ ansible-galaxy collection list
```

### Configuration

```console (paste below)
$ ansible-config dump --only-changed

```

### OS / Environment

_No response_

### Additional Information

_No response_

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by jatorcasso**
Depends-On: https://github.com/ansible-collections/amazon.aws/pull/856

##### SUMMARY
Fixes #843 

Can things like checking for a period after `description` be added to our sanity checks?

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
- aws_ax_info
- aws_caller_info
- aws_s3
- cloudformation_info
- ec2_eni_info
- ec2_group
- ec2_group_info
- ec2_instance_info
- ec2_key
- ec2_metadata_facts
- ec2_snapshot_info
- ec2_spot_instance
- ec2_spot_instance_info
- ec2_tag
- ec2_tag_info
- ec2_vpc_dhcp_option_info
- ec2_vpc_endpoint_info
- ec2_vpc_endpoint_service_info
- ec2_vpc_igw_info
- ec2_vpc_nat_gateway
- ec2_vpc_nat_gateway_info
- ec2_vpc_net_info
- ec2_vpc_route_table_info
- elb_classic_lb

### CODE REVIEW

**Comment by jatorcasso** [comment]
recheck

**Comment by jatorcasso** [comment]
recheck

**Comment by jatorcasso** [comment]
recheck

File: plugins/modules/ec2_vpc_dhcp_option_info.py
Comment (jillr) [inline plugins/modules/ec2_vpc_dhcp_option_info.py:102]:
```suggestion
            description: The aws resource id of the primary DHCP options set created or found.
```
This was in the source already, but while we're looking at it and cleaning up docs it would be nice to fix

**Review by jillr — APPROVED** [review]
Other than the typo LGTM - this should have a second review though because it's quite a lot of text to review

File: plugins/modules/aws_s3.py
Comment (tremble) [inline plugins/modules/aws_s3.py:13]: Sorry, #869 has just broken this.  I did however include it in #869

**Review by tremble — APPROVED** [review]
From experience, you may need to split out the changes to a couple of the slower tests to avoid timeouts.  We used to manually balance the groups of tests, and while the new automatic balancing is much better than it used to be slow tests can still cause problems.

**Comment by jatorcasso** [comment]
recheck

**Pull request merged**
